airbyte-integrations/connectors/destination-dev-null/src/test/resources/test-catalog.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
@edgao you aren't using these anywhere right? These are just left over from my dumb micronaut noodling?

Reply:
Correct, not using these